runtime/onert/core/src/compiler/pass/ConstantInsertionPass.cc

```diff
@@ -39,9 +39,10 @@ void ConstantInsertionPass::callback(const ir::OperationIndex &node_index, ir::I
   {
     auto &object = _graph.operands().at(input);
 
-    if (object.isConstant() && object.getUses().size() >= 2)
+    const auto key = ReplaceKey{input, factor};
+    if (object.isConstant() && (object.getUses().size() >= 2 ||
+                                _replace_operands_map.find(key) != _replace_operands_map.end()))
```

Comment:
This means replacing an operand index with other index although the corresponding operand is used once or less(zero).
Could you let me know why that operand should be replaced with a new operand in that case?

Reply:
Sorry for confusion. As we talked offline, you are right. Logically, the last remaining operand should be replaced as well.

Reply:
This function seeks to find all operands that use the same key value and change them to alternate operands. Howeer, it is executed only when the use is 2 or more, and the last remaining operand is not processed. In this case, there are two operands with the same value. This commit allows the last operand to use the same key if the same key has already been added.